packages/java/parser-jvm-plugin-model/src/main/java/dev/hilla/parser/plugins/model/ModelPlugin.java

```diff
@@ -66,6 +69,13 @@ public void enter(NodePath<?> nodePath) {
         }
 
         var schema = typedNode.getTarget();
+        if (signature instanceof BaseSignatureModel) {
+            schema.addExtension(JAVA_TYPE_KEY,
```

Comment:
Will this leak the class name for custom beans into the client side? Can we avoid that somehow?

Reply:
We could include-list the types that we explicitly support with column renderers / input field types.

Reply:
I think we should limit it for now BUT we will want the information later, in some form, to know e.g that ”company” in the Person class refers to the Company class

Reply:
At least for the schema that information is already there, so we could expose it in the generator when necessary.

Reply:
So you are saying we should include everything in openapi.json, which is not published, but selectively include in generated TS?

Reply:
For now I've added filtering for the Java types so that only primitives and any type from `java.**` will be included in the schema.